src/Microsoft.ML.AdditionalLearners/doc.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+    
+    <member name="OLS">
```

Comment:
Should we also remove the doc from

[https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.StandardLearners/Standard/doc.xml#L72](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.StandardLearners/Standard/doc.xml#L72)